source/Octodiff/Core/DeltaApplier.cs

```diff
@@ -16,34 +17,36 @@ public DeltaApplier()
 
         public void Apply(Stream basisFileStream, IDeltaReader delta, Stream outputStream)
         {
+            var buffer = new byte[bufferLength];
+
             delta.Apply(
-                writeData: (data) => outputStream.Write(data, 0, data.Length),
-                copy: (startPosition, length) =>
+                writeData: outputStream.Write,
+                copy: (offset, count) =>
                 {
-                    basisFileStream.Seek(startPosition, SeekOrigin.Begin);
+                    basisFileStream.Seek(offset, SeekOrigin.Begin);
 
-                    var buffer = new byte[4*1024*1024];
                     int read;
                     long soFar = 0;
-                    while ((read = basisFileStream.Read(buffer, 0, (int)Math.Min(length - soFar, buffer.Length))) > 0)
+                    while ((read = basisFileStream.Read(buffer, 0, (int)Math.Min(count - soFar, buffer.Length))) > 0)
```

Comment:
Nit: Use the `const` here rather than evaluating the buffer length.

Reply:
Thanks Adam, this has been addressed [here](https://github.com/OctopusDeploy/Octodiff/pull/39/commits/5b886de05dd2fa0ca1118cdefbc894ed6652e693).